SimpleServlet/src/main/java/com/skankhunt220/api/controller/RequestHandler.java

```diff
@@ -0,0 +1,76 @@
+package main.java.com.skankhunt220.api.controller;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.bson.Document;
+
+import main.java.com.skankhunt220.entities.User;
+import main.java.com.skankhunt220.service.UserService;
+
+@WebServlet("/Example") // servlet url
+public class RequestHandler extends HttpServlet {
```

Comment:
In my opinion, you have a wrong named controller. I don't have the right rules for naming a controller, but I think good practise using the suffix `Controller` to understand what this class present. Your controller works with the `User` entity, so `UserController` seems appropriate to me.